### ISSUE TITLE
Make the Site Accessible

### ISSUE THREAD

**Issue by jonathanhuydieu** [ucf-senior-design/senior-design#15]
Accessibility is an important aspect of every consumer application. It should be accommodating for users that may have issues with color blindness or other visual impairments in order to ensure a comfortable experience for all users.
